model_hub/model_hub/mmdetection/_trial.py

```diff
@@ -61,16 +62,15 @@ def __init__(self, context: det_torch.PyTorchTrialContext) -> None:
         # We will control how data is moved to GPU.
         self.context.experimental.disable_auto_to_device()
 
+        dist.init_process_group(backend="gloo")
+
         # Build model and make sure it's compatible with horovod.
-        self.model = mmdet.models.build_detector(self.cfg.model)
-        # We need to monkey patch the reduce_mean function from mmdet.core.utils so that it uses
-        # horovod's allreduce primitive instead of pytorch distributed's all_reduce function.
-        utils.monkey_patch_reduce_mean(self.model)
-        # We need to convert pytorch SyncBatchNorm layers to horovod's.
-        utils.convert_syncbn_model(self.model)
+        model = mmdet.models.build_detector(self.cfg.model)
 
         # Initialize model
-        self.model.init_weights()
+        model.init_weights()
+        self.model = torch.nn.parallel.DistributedDataParallel(model)
```

Comment:
Why don't we wrap the model in DistributedDataParallel on behalf of the user?  That would be more inline with our existing api, where we wrap the user's optimizer for them, only in dtrain scenarios

Reply:
done